src/main/scala/cromwell/binding/expression/FileEvaluator.scala

```diff
@@ -78,9 +78,15 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
    */
   private def evaluateRecursive(ast: AstNode): Try[Seq[WdlFile]] = {
     ast match {
+      case a: Ast if a.isGlobFunctionCall =>
+        evalValueToWdlFile(a.params.head) match {
+          case Success(WdlFile(value, _)) => Success(Seq(WdlFile(value, GlobOrNot.YesGlob)))
+          case Failure(e) => Failure(e)
+        }
+
       case a: Ast if a.isFunctionCallWithOneFileParameter =>
         evalValueToWdlFile(a.params.head) match {
-          case Success(v) => Success(v)
+          case Success(v) => Success(Seq(v))
```

Comment:
Yeah after I wrote that I realized that I avoid the map usually for the same reason. And then I hate myself a little bit.